crates/spicepod/src/component/view.rs

```diff
@@ -21,7 +21,7 @@ use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
 
-use super::{Nameable, WithDependsOn};
+use super::{Nameable, WithDependsOn, dataset::acceleration};
```

Comment:
Now that we're reusing the acceleration struct, let's move it to spicepod/src/acceleration/mod.rs

Reply:
Moved to `spicepod/src/acceleration/mod.rs`, thank you